.code-samples.meilisearch.yaml

```diff
@@ -450,7 +443,8 @@ getting_started_update_searchable_attributes: |-
   ])
 getting_started_update_stop_words: |-
   client.index('movies').updateStopWords(['the'])
-PLEASE_UPDATE_ME>>>>>>>>>>>>>PLEASE_UPDATE_ME>>>>>>>>>>>>>getting_started_check_task_status: |-
+? PLEASE_UPDATE_ME>>>>>>>>>>>>>PLEASE_UPDATE_ME>>>>>>>>>>>>>getting_started_check_task_status
+: |-
```

Comment:
My linter automatically saves to this. Please ignore :')

Reply:
Did you know you could interactively add chunks of code to the commit?

use `git add -p`

Reply:
I thought about it just after. Normally I do it but I completely forgot to do it